purchases/src/test/java/com/revenuecat/purchases/attributes/SubscriberAttributesPurchasesTests.kt

```diff
@@ -172,8 +76,6 @@ class SubscriberAttributesPurchasesTests {
 
     @After
     fun tearDown() {
-        postReceiptError = null
-        postReceiptCompletion = null
```

Comment:
I noticed there was a ton of unused code here after I removed the tests (since they were moved). So took the opportunity to remove it.